doc/content/getting-started/migrating/migrating-between-tts/establish-new-session/_index.md

```diff
@@ -70,6 +85,8 @@ Next, you need to import the `device-description.json` file in your {{% tts %}}
 
 {{< /tabs/container >}}
 
+If you migrated devices using the [migration tool]({{< ref "/getting-started/migrating/migration-tool" >}}) the process of migrating OTAA devices ends here. Otherwise follow the last steps below:
```

Comment:
```suggestion
If you migrated devices using the [migration tool]({{< ref "/getting-started/migrating/migration-tool" >}}) the process of migrating OTAA devices ends here. Otherwise keep following steps below.
```